src/util/mp_arith.h

```diff
@@ -11,11 +11,8 @@ Author: Daniel Kroening, kroening@kroening.com
 #define CPROVER_UTIL_MP_ARITH_H
 
 #include <string>
-#include <iosfwd>
```

Comment:
But there is use of `std::ostream`? Presumably some other header currently drags this in, but removing this include won't be beneficial?

Reply:
Failing CI suggests that it might not actually get dragged in elsewhere...